include/fastdds_statistics_backend/exception/Exception.hpp

```diff
@@ -292,9 +292,35 @@ class CorruptedFile : public Exception
             const CorruptedFile& other) = default;
 };
 
+/**
+ * @brief Exception to signal that there has been found an inconsistency inside the database.
+ */
+class Inconsistency : public Exception
+{
+
+public:
+
+    using Exception::Exception;
```

Comment:
This is done differently on the rest of the exceptions declared on this header. I think we should be consistent. I like this way better (`using Exception::Exception` to let the compiler automatically replicate the constructors in the parent class), so I would change the other classes, but I prefer having @EduPonz opinion on this.

Reply:
I also like this one better, and changing the others does not seem like a big time-consuming effort, so we might as well take the oportunity. It'll reduce the cpp, which is always nice.